### PR TITLE
Add Quarkus test suite PR template

### PR DESCRIPTION
**Pull request by pjgg**
### Summary

Add pull request template to Quarkus test suite main branch

### Checklist:
- [X] Commits are well encapsulated and follow [team agreetments](https://gitlab.cee.redhat.com/quarkus-qe/signpost/-/blob/main/pull-requests.md)

### CODE REVIEW

**Review by rsvoboda — CHANGES_REQUESTED** [review]
Same comment as for https://github.com/quarkus-qe/quarkus-test-framework/pull/438#pullrequestreview-916929960

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Summary
+
+(Summarize the problem solved by this PR, and how to verify it manually)
+
+Please delete options that are not relevant.
```
Comment (mjurc) [inline .github/pull_request_template.md:5]: Please select the relevant options

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New scenario (non-breaking change which adds functionality)
+- [ ] This change requires a documentation update
+
+### Checklist:
+- [ ] Methods and classes used in PR scenarios are meaningful according to the purpose of this test
```
Comment (mjurc) [inline .github/pull_request_template.md:12]: I'm not quite sure what this means

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+- [ ] External tickets are updated according to these changes
+- [ ] Commits are well encapsulated and follow [team agreetments](https://gitlab.cee.redhat.com/quarkus-qe/signpost/-/blob/main/pull-requests.md)
```
Comment (mjurc) [inline .github/pull_request_template.md:14]: I don't really like this for multitude of reasons:

* this is an opensource project, but the doc is closed behind VPN
* we're exposing internal infra 
* we have contributors not in the team

**Review by mjurc — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+Please select the relevant options.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Dependency update
+- [ ] New scenario (non-breaking change which adds functionality)
+- [ ] This change requires a documentation update
+
+### Checklist:
+- [ ] Methods and classes used in PR scenarios are meaningful
+- [ ] Commits are well encapsulated and follow [good practices](https://cbea.ms/git-commit/)
```
Comment (mjurc) [inline .github/pull_request_template.md:14]: I think the phrase is 'the best practices', right?

**Review by mjurc — CHANGES_REQUESTED** [review]

**Review by mjurc — COMMENTED** [review]
I'm leaving a comment here as this is hanging in my review requests.

**Pull request merged**